extensions/logging-gelf/runtime/src/main/java/io/quarkus/logging/gelf/GelfConfig.java

```diff
@@ -61,4 +64,28 @@
      */
     @ConfigItem(defaultValue = "yyyy-MM-dd HH:mm:ss,SSS")
     public String timestampPattern;
+
+    /**
+     * The logging-gelf log level.
+     */
+    @ConfigItem(defaultValue = "ALL")
+    public Level level;
+
+    /**
+     * Name of the Facility.
+     */
+    @ConfigItem(defaultValue = "jboss-logmanager")
+    public String facility;
+
+    /**
+     * Post additional fields.
+     * You can add static fields to each log event in the following form:
+     * ```
+     * quarkus.log.handler.gelf.additional-field.field1.value=value1
+     * quarkus.log.handler.gelf.additional-field.field1.type=String
+     * ```
+     */
+    @ConfigItem
```

Comment:
Please add a `@ConfigDocMapKey("field-name")` annotation.

Reply:
I was not aware of this annotation, updated to use it.

Reply:
I also add a `@ConfigDocSection` it adds clarity to the configuration documentation.